Stb isolation (#4192)
mlpack/mlpack: doc/user/install.md

| `-DUSE_OPENMP=ON` | Use OpenMP for parallelization. | `ON` |
| `-DUSE_PRECOMPILED_HEADERS=OFF` | Disable precompiled headers during build. | `OFF` |
| `-DUSE_SYSTEM_STB=OFF` | Use version of STB bundled with mlpack. If set to `ON` make sure `stb_image.h`, `stb_image_write.h`, and `stb_image_resize2.h` are available. | `OFF` |

> | `-DDISABLE_STB=OFF` | Disable stb support within mlpack; use this if your system does not need image processing (e.g., real time sensors data). | `OFF ` |

Reviewer: ```suggestion
| `-DDISABLE_STB=OFF` | Disable STB support within mlpack; use this to reduce program size if you do not need image processing (e.g., you are only using real-time sensor data). | `OFF ` |
```

| `-DUSE_SYSTEM_DR_LIBS=OFF` | Use the version of dr\_mp3 and dr\_wav available on the system instead of the version bundled with mlpack.  If set, make sure `dr_mp3.h` and `dr_wav.h` are available. | `OFF` |

> | `-DDISABLE_DR_LIBS=OFF` | Disable dr\_libs (audio) support within mlpack; use this if your system does not need audio processing (e.g., real time sensors data). | `OFF` |

Reviewer: ```suggestion
| `-DDISABLE_DR_LIBS=OFF` | Disable dr_libs (audio) support within mlpack; use this if you do not need audio processing (e.g., you are only using real-time sensor data). | `OFF` |
```
I don't think the `\` is needed... might be worth double-checking the compiled version after accepting this suggestion to see.
